lib/rubocop/cop/style/redundant_line_continuation.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Check for redundant line continuation.
+      #
+      # This cop is marked as redundant if removing the backslash
+      # does not result in a syntax error.
+      # However, a backslash at the end of a comment or
+      # for string concatenation is not redundant and is not considered an offense.
+      #
+      # @safety
+      #   This cop is unsafe, as it can produce false positives if this cop is
+      #   checks for a syntax error when a backslash is removed.
+      #   However, the following cases are false positives because remove a backslash
+      #   does not result in a syntax error.
+      #
+      #   [source,ruby]
+      #   ----
+      #   1 \
+      #     + 2 \
+      #       - 3
+      #   some_method \
+      #     (argument)
+      #   ----
```

Comment:
This cop should be implemented so that false positives do not occur as a safe cop. So this case should be allowed.

Reply:
I see, I have updated this PR to avoid false positives for those cases.